bin/run-librivox.sh

```diff
@@ -5,12 +5,18 @@ if [ ! -f DeepSpeech.py ]; then
     exit 1
 fi;
 
+
+
 python -u DeepSpeech.py \
   --importer librivox \
-  --train_batch_size 32 \
-  --dev_batch_size 32 \
-  --test_batch_size 32 \
+  --train_batch_size 12 \
+  --dev_batch_size 12 \
+  --test_batch_size 12 \
   --learning_rate 0.0001 \
-  --epoch 50 \
+  --epoch 15 \
   --display_step 5 \
+  --validationstep 5 \
+  --dropout_rate 0.30 \
+  --default_stddev 0.046875 \
+  --checkpoint_step 1 \
```

Comment:
This no no longer handled the same way as before. Checkpoints are done now at a time interval. See for example[[1](https://github.com/mozilla/DeepSpeech/blob/master/DeepSpeech.py#L104)]
```python
...
tf.app.flags.DEFINE_integer ('checkpoint_secs', 600, 'checkpoint saving interval in seconds')
...
```